lib/main.dart

```diff
@@ -1,12 +1,112 @@
 import 'package:flutter/material.dart';
+import 'data.dart';
 
 void main() => runApp(MyApp());
 
 class MyApp extends StatelessWidget {
-  @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      home: Center(child: Text('Empty Shell for Art Museum')),
+      theme: ThemeData(primaryColor: Colors.teal[50]),
+      home: HomePage(),
+    );
+  }
+}
+
+class HomePage extends StatefulWidget {
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  List<ObjectInfo> picData;
+
+  void initState() {
+    super.initState();
+    requestImages().then((images) {
+      setState(() => picData = images);
+    });
+  }
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.teal[100],
+      appBar: AppBar(
+        title: Text('The Art Museum'),
+        backgroundColor: Colors.teal[50],
+      ),
+      body: Column(
```

Comment:
You may want to add a SafeArea here, I noticed there is some clipping on some devices (shown here at the bottom):
![Screen Shot 2019-08-01 at 10 31 40 AM](https://user-images.githubusercontent.com/16964204/62314351-a70f8280-b447-11e9-8ac7-fc3ee8444849.png)